.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -14,6 +14,12 @@ body:
   - type: markdown
     attributes:
       value: 'next@canary is the beta version of Next.js. It includes all features and fixes that are pending to land on the stable release line.'
+  - type: textarea
+    attributes:
+      label: Run `next info` (available from version 12.0.8 and up)
```

Comment:
I think suggesting `npx next info` is better because it doesn't require the `next` executable in PATH. More skilled users will understand that they can run `yarn next info` or such

Reply:
Suggesting that in the description. That is just the title